server/models/handlers/account.js

```diff
@@ -2,8 +2,20 @@
 the front end UI in order to create a user account in the backend*/
 
 async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
-    
-    const user = {
+     // Searches the student table to see if a student account exists for the given data.
+     const user = await studentModel.insertStudentAccount(firstName, lastName, type, studentID, email, password, phone);
```

Comment:
Hi Nancy,
Currently, this inserts ANY user into both tables. We only want to insert the user into the appropriate table. Thus, I would recommend checking the userType and based on the userType, call the appropriate insert method. For example, `if userType === UserType.student` then call the method in line 6. You can import the constant values from the file userTypes.json in the constants folder.

Reply:
Hi Amy,
Good catch!
My action: I've added a condition which inserts the student user into the student table and the worker user into the worker table.
You can now see these changes in:
line 7-14 of server/models/handlers>account.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy, 
Wonderful! Thank you for making those changes.